Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -412,23 +412,53 @@ final class BuildCommandTests: CommandsTestCase {
         }
     }
 
-    func testXcodeBuildSystemDefaultSettings() async throws {
-        #if !os(macOS)
-        try XCTSkipIf(true, "test requires `xcbuild` and is therefore only supported on macOS")
-        #endif
+    private func testBuildSystemDefaultSettings(buildSystem: String) async throws {
         try await fixture(name: "ValidLayouts/SingleModule/ExecutableNew") { fixturePath in
             // try await building using XCBuild with default parameters.  This should succeed.  We build verbosely so we get
             // full command lines.
-            let defaultOutput = try await execute(["-c", "debug", "-v"], packagePath: fixturePath).stdout
+            let output = try await execute(["--build-system", buildSystem, "-c", "debug", "-v"], packagePath: fixturePath)
 
-            // Look for certain things in the output from XCBuild.
+            // In the case of the native build system check for the cross-compile target, only for macOS
+#if os(macOS)
```

Comment:
This indicates in the change of behavior in llbuild, the `-target` option was previously passed on all platforms, not just macOS. What's the reason for the change then?